jobs/tools/write_cosmo_input_ghg.py

```diff
@@ -107,8 +107,9 @@ def main(csv_filename, namelist_filename, cfg=None):
     """
 
     #Distinguish between COSMO and ICON
-    model = "COSMO"
-    if cfg.target is tools.Target.ICON or cfg.target is tools.Target.ICONART:
+    if cfg.model.startswith('cosmo'):
+        model = "COSMO"
+    if cfg.model.startswith('icon'):
         model = "ICON"
```

Comment:
Does it happen that `write_cosmo_input_ghg` is called for ICON? Is it the correct function name in that case?

Reply:
Yes, this function is also called for ICON. I will rename it accordingly.

Reply:
Ok apparently it is not used in ICON (anymore). I removed the parts within the function. The name can stay like that.